.rubocop.yml

```diff
@@ -0,0 +1,9 @@
+AllCops:
+  NewCops: enable
+  TargetRubyVersion: 2.6
```

Comment:
Is this right?

Reply:
Yes, it matches the ruby version we specified in the gemspec. We could update to `3.0` but I think 2.x is safe since it's what we've been using until now (and the changes I made are compatible with that version)